lib/workflow_metal_postgres_adapter/schema/transition.ex

```diff
@@ -8,14 +8,21 @@ defmodule WorkflowMetalPostgresAdapter.Schema.Transition do
   alias WorkflowMetalPostgresAdapter.Schema.ExecutorType
   alias WorkflowMetal.Storage.Schema.Transition
 
-  import EctoEnum
-  defenum(JoinType, :"#{@prefix}_transition_join_type", [:none, :and], schema: @schema)
-  defenum(SplitType, :"#{@prefix}_transition_split_type", [:none, :and], schema: @schema)
+  alias WorkflowMetalPostgresAdapter.Schema.TransitionTypesForTestAndExample.{
+    SplitType,
+    JoinType
+  }
+
+  @config Application.compile_env(:workflow_metal_postgres_adapter, WorkflowMetalPostgresAdapter)
+  @enum_types Keyword.get(@config, :enum_types, [])
+
+  @split_type get_in(@enum_types, [:transition, :split_type]) || SplitType
+  @join_type get_in(@enum_types, [:transition, :join_type]) || JoinType
```

Comment:
为什么会套一层 WorkflowMetalPostgresAdapter 呢？
```suggestion
  @config Application.compile_env!(:workflow_metal_postgres_adapter, [WorkflowMetalPostgresAdapter, :enum_types, :transition, :split_type])
  @config Application.compile_env!(:workflow_metal_postgres_adapter, [WorkflowMetalPostgresAdapter, :enum_types, :transition, :join_type])
```

Reply:
compile_env 和 get_env 的参数有点不一样